src/overview/reducer.js

```diff
@@ -5,13 +5,16 @@ import * as actions from './actions'
 const defaultState = {
     searchResult: {rows: []},
     query: '',
+    isFetching: true,
 }
 
 function setQuery(state, {query}) {
+	state.isFetching = true; //will start fetching
     return {...state, query}
 }
 
 function setSearchResult(state, {searchResult}) {
+	state.isFetching = false; //will stop fetching
```

Comment:
What i mean was when we call setSearchResult means the database has returned a value now if the query is changed and we again make it true